File: 260.cpp
class Solution {
public:
    vector<int> singleNumber(vector<int>& nums) {
        vector<int> ans;
        int pointer=0;
        for (int i=0;i<nums.size();i++)
        {
    		
            for (int j=0;j<nums.size();j++)
            {
        		if (j==i && j!= nums.size() - 1) {continue;}
    			if (nums[j] == nums[i] && j!=i)
                {
                    break;
                }
    			if (j == nums.size()-1)
                {
    				ans.push_back(nums[i]);
                }
            }
    	}
        return ans;
    }
};

File: 260/260.java
class Solution {
    public int[] singleNumber(int[] nums) {
        int xor = 0;
        int[] ans = new int[2];
        for (int num:nums){
            xor ^= num;
        }
        xor &= ~(xor&(xor-1));
        for (int num:nums){
            if ((num & xor) == 0){
                ans[0] ^= num;
            }else{
                ans[1] ^= num;
            }
        }
        return ans;
    }
}
